pkgs/development/libraries/libsixel/default.nix

```diff
@@ -0,0 +1,31 @@
+{stdenv, fetchFromGitHub}:
+stdenv.mkDerivation rec {
+  version = "1.6.1";
+  name = "libsixel-${version}";
+
+  src = fetchFromGitHub {
+    repo = "libsixel";
+    rev = "ef4374f80385edc48e0844cf324d7ef757688e44";
+    owner = "saitoha";
+    sha256 = "08m5q2ppk235bzbwff1wg874vr1bh4080qdj26l39v8lw1xzlqcp";
+  };
+
+  meta = with stdenv.lib; {
+    description = "The SIXEL library for console graphics, and converter programs";
+    homepage = http://saitoha.github.com/libsixel;
+    maintainers = with maintainers; [ vrthra ];
+    license = ''
```

Comment:
This is kmiya's sixel license.
https://github.com/saitoha/sixel/

The license of libsixel is MIT.
https://github.com/saitoha/libsixel#license

Certainly, libsixel is derived from kmiya's sixel, but I had relicensed its license to MIT with permission of kmiya.